concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+248	Bowern-2012-348-248	see	1409	SEE	
+249	Bowern-2012-348-249	sharp	1396	SHARP	
+250	Bowern-2012-348-250	shin	186	SHIN	
+251	Bowern-2012-348-251	shoot	1172	SHOOT	
+252	Bowern-2012-348-252	short	1645	SHORT	
+253	Bowern-2012-348-253	shoulder	1482	SHOULDER	
+254	Bowern-2012-348-254	show	1039	SHOW	
+255	Bowern-2012-348-255	shy/ashamed	487	SHY	
+256	Bowern-2012-348-256	sing	1261	SING	
+257	Bowern-2012-348-257	sit	1416	SIT	
+258	Bowern-2012-348-258	skin	763	SKIN	
+259	Bowern-2012-348-259	sky	1732	SKY	
+260	Bowern-2012-348-260	sleep	1585	SLEEP	
+261	Bowern-2012-348-261	slow	701	SLOW	
+262	Bowern-2012-348-262	small	1246	SMALL	
+263	Bowern-2012-348-263	smoke	778	SMOKE (EXHAUST)	
+264	Bowern-2012-348-264	snake	730	SNAKE	
+265	Bowern-2012-348-265	sneeze	1621	SNEEZE	
+266	Bowern-2012-348-266	sniff/smell	1586	SMELL (PERCEIVE)	
+267	Bowern-2012-348-267	son	1620	SON	
+268	Bowern-2012-348-268	south	1595	SOUTH	
+269	Bowern-2012-348-269	speak	1623	SPEAK	
+270	Bowern-2012-348-270	spear	945	SPEAR	
+271	Bowern-2012-348-271	spearthrower	2677	SPEARTHROWER	
+272	Bowern-2012-348-272	spider	843	SPIDER	
+273	Bowern-2012-348-273	spit	1440	SPIT	
+274	Bowern-2012-348-274	split	1437	SPLIT	
+275	Bowern-2012-348-275	squeeze	1414	SQUEEZE	
+276	Bowern-2012-348-276	stab/pierce	1434	STAB	
+277	Bowern-2012-348-277	stand	1442	STAND	
+278	Bowern-2012-348-278	star	1430	STAR	
+279	Bowern-2012-348-279	steal	713	STEAL	
+280	Bowern-2012-348-280	stick/wood	1295	STICK	
+281	Bowern-2012-348-281	stingray	1910	STINGRAY	
+282	Bowern-2012-348-282	stone	857	STONE	
+283	Bowern-2012-348-283	suck	1421	SUCK	
+284	Bowern-2012-348-284	sun	1343	SUN	
+285	Bowern-2012-348-285	swan	2837	SWAN	
+286	Bowern-2012-348-286	sweat	2458	PERSPIRE OR SWEAT	
+287	Bowern-2012-348-287	swell	1573	SWELL	
+288	Bowern-2012-348-288	swim	1439	SWIM	
+289	Bowern-2012-348-289	tail	1220	TAIL	
+290	Bowern-2012-348-290	take something	1749	TAKE	
+291	Bowern-2012-348-291	tendon	1137	TENDON	
+292	Bowern-2012-348-292	that	78	THAT	
+293	Bowern-2012-348-293	thatch/roof	2100	THATCH OR ROOF	
+294	Bowern-2012-348-294	they	817	THEY	
+295	Bowern-2012-348-295	thick	1244	THICK	
+296	Bowern-2012-348-296	thigh	800	THIGH	
+297	Bowern-2012-348-297	thin	2308	THIN	
+298	Bowern-2012-348-298	think	2271	THINK	
+299	Bowern-2012-348-299	thirsty	847	THIRSTY	
+300	Bowern-2012-348-300	this	1214	THIS	
+301	Bowern-2012-348-301	thou	2844	YOU (FORMAL ADDRESS)	Or is it plural?
+302	Bowern-2012-348-302	three	492	THREE	
+303	Bowern-2012-348-303	throat	1346	THROAT	
+304	Bowern-2012-348-304	throw	1456	THROW	
+305	Bowern-2012-348-305	thumb	1781	THUMB	
+306	Bowern-2012-348-306	thunder	1150	THUNDER	
+307	Bowern-2012-348-307	tie up/fasten	1094	FASTEN	
+308	Bowern-2012-348-308	tobacco	974	TOBACCO	
+309	Bowern-2012-348-309	together	2482	TOGETHER	
+310	Bowern-2012-348-310	tomorrow	1329	TOMORROW	
+311	Bowern-2012-348-311	tongue	1205	TONGUE	
+312	Bowern-2012-348-312	tooth	1380	TOOTH	
+313	Bowern-2012-348-313	top grinding stone	3027	TOP GRINDSTONE	
```

Comment:
Did you add the concept TOP GRINDSTONE? If so, it is probably narrower as relation that GRINDSTONE, right? This should be added.

Reply:
I did, the conceptlist has both TOP GRINDSTONE and BOTTOM GRINDSTONE. I was about the code the top grindstone as GRINDSTONE and the bottom one as either STONE or (adding the concept of) BOTTOM GRINDSTONE, don't exactly recalled why I decided to do it this way.

Will check the relation.